shared/uavobjectdefinition/stabilizationsettings.xml

```diff
@@ -27,6 +27,9 @@
 		<field name="PitchPI" units="" type="float" elementnames="Kp,Ki,ILimit" defaultvalue="2.5,0,50" limits="%BE:0:20,%BE:0:20,"/>
 		<field name="YawPI" units="" type="float" elementnames="Kp,Ki,ILimit" defaultvalue="2.5,0,50" limits="%BE:0:20,%BE:0:20,"/>
 		<field name="GyroCutoff" units="Hz" type="float" elements="1" defaultvalue="55.0"/>
+		<field name="GyroButterworth" units="" type="enum" elements="1" options="FALSE,TRUE" defaultvalue="FALSE">
```

Comment:
(brick wall at 25Hz instead of a sloping filter at 12Hz like we have now or a 2nd order at 18Hz or whatever).